dumper/app/src/test/java/com/google/edwmigration/dumper/application/dumper/MetadataDumperTest.java

```diff
@@ -16,23 +16,76 @@
  */
 package com.google.edwmigration.dumper.application.dumper;
 
+import com.google.edwmigration.dumper.application.dumper.connector.Connector;
 import com.google.edwmigration.dumper.application.dumper.connector.bigquery.BigQueryLogsConnector;
+import com.google.edwmigration.dumper.application.dumper.connector.hive.HiveMetadataConnector;
+import java.io.File;
+import org.junit.AfterClass;
+import org.junit.Assert;
 import org.junit.Test;
 import org.junit.runner.RunWith;
 import org.junit.runners.JUnit4;
 
-/**
- *
- * @author shevek
- */
+/** @author ishmum */
 @RunWith(JUnit4.class)
 public class MetadataDumperTest {
 
-    @Test
-    public void testInstantiate() throws Exception {
-        MetadataDumper dumper = new MetadataDumper();
-        // dumper.run("--help");    // Calls System.exit(1)
-        dumper.run("--connector", new BigQueryLogsConnector().getName(), "--dry-run");
-    }
+  private static File FILE;
```

Comment:
A different file is created in each test. I would suggest to remove the static modifier and delete the file in the non-static `@After` method instead of `@AfterClass` one.